core/app/models/spree/order.rb

```diff
@@ -400,7 +400,7 @@ def deliver_order_confirmation_email
 
     # Helper methods for checkout steps
     def paid?
-      payment_state == 'paid' || payment_state == 'credit_owed'
+      %w(paid credit_owed).include?(payment_state)
```

Comment:
We can definitely agree that this isn't quite the appropriate place for this discussion. It does sound like you have some specific rules/guidelines/axioms you'd like the project to adopt, would you consider opening up an issue for them?

Though it would be great if anyone else wanted to weigh in on the "when to do a global replace" discussion. I don't consider it a "feels" decision so much as a pragmatic call that doesn't really warrant enough investigation or decision making to warrant any kind of "scientific" investigation, but am curious on other opinions.